src/uwtools/tests/drivers/test_chgres_cube.py

```diff
@@ -101,6 +102,7 @@ def driverobj(config, cycle):
         "_scheduler",
         "_validate",
         "_write_runscript",
+        "output",
```

Comment:
This is added in many of the driver test modules that follow. This states that the `output` method is inherited from `Driver`. When a driver module eventually overrides the default, exception-raising `output` inherited from `Driver`, this test will fail and this line can be removed to make it pass.